rohd_devtools_extension/lib/main.dart

```diff
@@ -1,18 +1,32 @@
-// Copyright (C) 2024 Intel Corporation
+// Copyright (C) 2025 Intel Corporation
```

Comment:
update copyrights with additional year, rather than replace (i.e. use `2024-2025`) -- also make sure to update in all files touched